virtool/users/pg.py

```diff
@@ -28,7 +28,7 @@ class SQLUser(Base):
     b2c_given_name: Mapped[str] = ""
     b2c_family_name: Mapped[str] = ""
     b2c_oid: Mapped[str] = ""
-    force_reset: Mapped[bool] = False
+    force_reset: Mapped[bool] = Column(Boolean, default=False)
```

Comment:
Was the old line not working? I am a bit surprised. I thought it was valid new-style model definition using `Mapped`. It would like to figure out why it doesn't work.

This is work a try, but I don't want to do it without being clear on why a simple `= False` is not working for the default value.
```suggestion
    force_reset: Mapped[bool] = mapped_column(default=False)
```

Reply:
The old line wasn't working for test_force_reset as the SQLUser did not store force reset without it being a column or mapped_column. See picture below, and note that all b2c is not stored in the SQLUser as well.

The Mapped[bool] is only a type hint that is really only useful to the reader, unless it gets checked with mypy or other static analysis tools.

The mapped_column does work to pass the test and not fail any other existing test.

![image](https://github.com/virtool/virtool/assets/63571383/4ec34478-d43f-4aa1-9266-5c817f6823dd)